geonode/management_commands_http/forms.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django import forms
+
+from geonode.management_commands_http.models import ManagementCommandJob
+from geonode.management_commands_http.utils.commands import (
+    get_management_commands,
+)
+
+
+class ManagementCommandJobAdminForm(forms.ModelForm):
+    autostart = forms.BooleanField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.available_commands = get_management_commands()
+        self.fields["command"] = forms.ChoiceField(
+            choices=[(command, command) for command in self.available_commands]
+        )
+
+    class Meta:
+        model = ManagementCommandJob
+        fields = ("command", "args", "kwargs",)
+
+    def clean_args(self):
+        value = self.cleaned_data.get('args')
+        value_json = json.loads(value)
+        if type(value_json) != list:
+            self.add_error("args", 'args must be a list')
+
+        if "--help" in value_json:
+            self.add_error("args", 'Forbidden argument: "--help"')
+
+        return value
+
+    def clean_kwargs(self):
+        value = self.cleaned_data.get('kwargs')
+        value_json = json.loads(value)
```

Comment:
Consider to use [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval) instead of `json.loads`:
```python
Example 1:
input = '["acb"]'
json.loads -> ['acb']
ast.literal_eval -> ['acb']

Example2: input = "['acb']"
json.loads -> json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)
ast.literal_eval -> ['acb']
```